config/01-ce-router-defaults.conf.in

```diff
@@ -59,9 +59,19 @@ COMPLETED_JOB_EXPIRATION = 30
 REMOVE_CLAUSE_3 = (JobStatus == 4 && time() - EnteredCurrentStatus > $(COMPLETED_JOB_EXPIRATION)*3600*24)
 REMOVE_REASON_3 = strcat("job completed over ", $(COMPLETED_JOB_EXPIRATION), " days ago.")
 
+# Remove jobs that
+REMOVE_CLAUSE_4 = (JobUniverse == 5 && \
+                   JobStatus == 2 && \
+                   time() - JobCurrentStartExecutingDate > \
+                       maxWalltime isnt defined ? $(ROUTED_JOB_MAX_TIME)*60 : maxWalltime*60 )
```

Comment:
`defined` isn't a keyword in ClassAds; it will evaluate to `undefined`. I think you want this instead: `maxWalltime is undefined`

Reply:
Good catch. Fixed!